### PR TITLE
update speech version to 1.32.1, and add new metadata files for new vision java packages.

### PR DESCRIPTION
**Pull request by BrianMouncer**
update speech version to 1.32.1, and add new metadata files for new vision java
packages.

### CODE REVIEW

File: metadata/latest/client-sdk.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "Name": "client-sdk",
-  "Version": "1.31.0",
+  "Version": "1.32.1",
   "DirectoryPath": "",
   "ServiceDirectory": "NA",
   "Group": "com.microsoft.cognitiveservices.speech",
```
Comment (dargilco) [inline metadata/latest/client-sdk.json:6]: I think the name of this file is wrong (`client-sdk.json`). If you look at all other files in this folder, then have a name that is related to package name. Did this name come from some template example, and need to be updated?

Related, why is the value of 'name' here "client-sdk" and not "com.microsoft.cognitiveservices.speech" or similar?

Comment (BrianMouncer) [reply metadata/latest/client-sdk.json:6]: No, the speech sdk package name is actually "Client-sdk", and is the file setup by the docs team.
I am just updating the version in the existing file.

**Comment by BrianMouncer** [comment]
@dargilco @danieljurek 

This adds the metadata files for the new vision java packages.
I'm 90% confident in what is here, but the [docs](https://eng.ms/docs/products/azure-developer-experience/develop/sdk-docs#metadata-json-files) list some "optional" fields, and it would be great to get feedback about if we should add a value for "DirectoryPath:" or "Namespaces:".  What benefit does adding these gain us as far as docs content differences?

Thanks,

Brian.

File: metadata/latest/client-sdk.json (hunk 1 above)
Comment (dargilco) [reply metadata/latest/client-sdk.json:6]: @BrianMouncer I understand this is not related to your change, and it should not block this change, but it seems to me that we got something wrong somewhere to end up with this generic name "client-sdk". let's continue to discuss this on IM.

**Review by dargilco — APPROVED** [review]

**Comment by BrianMouncer** [comment]
If this is helpfull

image

**Review by jhakulin — APPROVED** [review]

File: metadata/preview/azure-ai-vision-common-internal.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "Name": "azure-ai-vision-common-internal",
```
Comment (jhakulin) [inline metadata/preview/azure-ai-vision-common-internal.json:2]: Forgot to comment earlier that, azure-ai-vision-common-internal is internal package and would not be required in reference API documentation.

Please remove this from the docs.

**Comment by danieljurek** [comment]
These changes look OK to me. Once the `azure-ai-vision-common-internal.json` file is removed we can merge. 

> and it would be great to get feedback about if we should add a value for "DirectoryPath:" or "Namespaces:". What benefit does adding these gain us as far as docs content differences?

`DirectoryPath` should be in the required fields. I added it to the docs. That value points to the path of the package in our `azure-sdk-for-*` repos. In your case leave it blank because the source code isn't in our `azure-sdk-for-*` repo. 

`Namespaces` lists namespaces for a given .NET package and is used to generate ToC entries for that. This is not useful for Java packages. Today that role is fulfilled by the `.txt` file which lists packages in an artifact.

**Comment by danieljurek** [comment]
After speaking with @BrianMouncer, we're keeping the `-internal` json. So this is good to merge.

**Review by danieljurek — APPROVED** [review]

File: metadata/preview/azure-ai-vision-common-internal.json (hunk 2 above)
Comment (danieljurek) [reply metadata/preview/azure-ai-vision-common-internal.json:2]: We're keeping this one.

**Pull request merged**